src/point_one/fusion_engine/messages/control.h

```diff
@@ -587,7 +603,22 @@ struct P1_ALIGNAS(4) ShutdownRequest : public MessagePayload {
 struct P1_ALIGNAS(4) StartupRequest : public MessagePayload {
   static constexpr MessageType MESSAGE_TYPE = MessageType::STARTUP_REQUEST;
   static constexpr uint8_t MESSAGE_VERSION = 0;
-  /** A bitmask of flags associated with the event. */
+
+  /**
+   * Start navigation engine if not running.
+   */
+  static constexpr uint64_t START_ENGINE = 0x0000000000000001;
+  /**
+   *  If a log is not being generated, start a new log.
```

Comment:
Extra space before if

Would it be helpful to have a bit to close the current log and start a new one atomically? Or optionally reuse this bit to do that if already logging? Might be useful to allow user-directed cutting of logs.